pandas/io/tests/json/test_json_norm.py

```diff
@@ -225,6 +225,59 @@ def test_nested_flattens(self):
 
         self.assertEqual(result, expected)
 
+
+    def test_json_normalise_fix(self):
+        # issue 14505
```

Comment:
put a comment explaining what this is. rename test -> test_json_normalize_errors